packages/dx-react-grid/src/plugins/table-filter-row.jsx

```diff
@@ -21,7 +21,6 @@ const getFilterTableCellTemplateArgs = (
 ) => ({
   ...params,
   column: params.tableColumn.column,
-  row: params.tableRow.row,
```

Comment:
@gsobolev I'm thinking there must be some specific reason (that I don't know) as to why you removed this. Would you please elaborate?

I'm asking because in my use case, it would be extremely useful to have the rest of the row, or really the ID of the row so that I can access another field in my data. I'm not keep on having formatting code in my react component but instead I'd like to pre-format all of my data before it gets to the grid. In this case, my "formatter" would simply choose another field from the data to display.

Any thoughts would be very useful! Thanks for this amazing library!

Reply:
Hi @dsibiski!
 
I am not sure I understand your concerns. The `getFilterTableCellTemplateArgs` is a helper which returns parameters only for the filter row cells. The filter row itself does not hold any data fields.
Please provide more details on your use case so I can understand your scenario.
Let's also continue this discussion as a separate issue rather than in this pull request.

Reply:
@gsobolev Thanks for the reply, sorry for confusing the code (thought I was looking at something else). I'll open up a proper issue and explain my use case better there.